library.js

```diff
@@ -136,6 +137,11 @@ Solr.connect = function () {
 		delete Solr.client;
 	}
 
+	Solr.config = {
+		...Solr.config,
+		secure: Solr.config.secure && Solr.config.secure === 'on'
+	}
+
```

Comment:
Do you need to update `engines` in `package.json` to require a newer node version? Just in case...

Reply:
FYI Node v8.3.0 required for spread operator usage on object literals.

Reply:
FYI there is currently no engine specification in the plugin's `package.json`. But it has a recommendation for NodeBB 1.11 which at the time the requirements were to use NodeJS LTS 8 (which is currently 8.15.1). And currently with NodeBB 1.12 the requirements say to use NodeJS LTS 10.

So not really sure if this is really necessary?

Reply:
Okey dokey :+1: Works for me.